sources/platform/integrations/programming/api.md

```diff
@@ -164,7 +178,32 @@ If you set up a webhook pointing to the Apify API, the Apify platform will autom
 Therefore, you need to make sure the token has sufficient permissions not only to set up the webhook, but also to perform the actual operation.
 
 :::tip
-
 Let's say you want to create a webhook that pushes an item to a dataset every time an Actor successfully finishes. Then such a scoped token needs to be allowed to both run the Actor (to create the webhook), and write to that dataset.
-
 :::
+
+### Troubleshooting
+
+#### How do I allow a token to run a task?
+
+There is no dedicated run permission for tasks. Instead, configure the token with the following permissions:
```

Comment:
@TC-MO The linter complains about `There is` :sweat_smile: Of course I could phrase it differently, but in this particular case it just feels the best...

![image](https://github.com/user-attachments/assets/34cb2dea-232a-45ee-8de0-ef9804a782d6)

Reply:
I'd rather not disable the rule completely as I agree with it, so I'll take a look if there is any rewrite that makes sense

If not we can utilize `<-- vale off -->` and `<-- vale on -->` to turn it of for this specific sentence